operator/controllers/keptntask/function_utils.go

```diff
@@ -44,7 +46,7 @@ func (r *KeptnTaskReconciler) generateFunctionJob(task *klcv1alpha1.KeptnTask, p
 
 	container := corev1.Container{
 		Name:  "keptn-function-runner",
-		Image: "ghcr.io/keptn-sandbox/functions-runtime:main.202209211413",
+		Image: os.Getenv("FUNCTION_RUNNER_IMAGE"),
```

Comment:
I'd suggest we gather all environment variables in the `envConfig` struct (see https://github.com/keptn-sandbox/lifecycle-controller/blob/03d234610fd8ef9f21e756450c7f503cb236f302/operator/main.go#L90) and retrieve them from there

Reply:
as discussed, this will be done in a follow up PR